### ISSUE TITLE
Separate node-watcher into 2 packages

### ISSUE THREAD

**Issue by amaurymartiny** [yjkimjunior/Nomidot#33]
Depends on: https://github.com/yjkimjunior/Nomidot/issues/20

The idea would be to have 2 packages:
- `@substrate/node-watcher`: This one will be published to npm, it will only hold what's currently in `back/node-watcher/src/nodeWatcher.ts`
- `@substrate/nomidot-watcher`: This one will hold the prisma server, and all the tasks that are relevant to Nomidot. Will use `@substrate/node-watcher` internally

The idea is to make `@substrate/node-watcher` reusable to other people. They would just install `@substrate/node-watcher`, and create their own tasks.

I just talked to Thibaut, there's a high probability that the governance platform will just use the same DB as we generate here with Nomidot (assuming we add 1-2 more tasks about governance). If that's the case, then this issue is low priority.
